workflow/rules/common.smk

```diff
@@ -60,13 +60,20 @@ def get_personal_data(wildcards):
     return expand(pattern, img=paths)
 
 
+def load_tsv(f):
+    try:
+        return pd.read_csv(f, sep="\n", header=None, squeeze=True)
+    except pd.errors.EmptyDataError:
+        return []
```

Comment:
I understand that the purpose is to return an empty data structure, to escape the error and the workflow can carry on, but wouldn`t an empty pandas data-frame be the adequate placeholder instead of a list, since pd.read_csv returns a dataframe as well?

Reply:
Interesting question! Later on, in Line 110, the return of this function is expanded over a path pattern. Snakemake can expand over both - lists and Series. There is no specific reason, why I chose a list - it just felt more natural to me to use an empty list, rather than a DataFrame or Series. However, an empty Series would probably also work.